src/shaders/arview.frag

```diff
@@ -11,13 +11,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#extension GL_OES_EGL_image_external : require
-
 precision mediump float;
 
 varying vec2 vTextureCoord;
 
-uniform samplerExternalOES uSampler;
+uniform sampler2D uSampler;
```

Comment:
Will this break on older installations of WebARonARCore? Wasn't the samplerExternalOES necessary?

Reply:
There are still shaders that use the External Image OES extension. WebARonARCore still uses this sampler type, just that the code is in a different shader file (with the -oes suffix).